src/QueueReceiver.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using Microsoft.EntityFrameworkCore;
+using QueueReceiver.Core.Interfaces;
 using QueueReceiver.Core.Models;
+using System.Threading.Tasks;
 using Person = QueueReceiver.Core.Models.Person;
 
 namespace QueueReceiver.Infrastructure.Data
 {
-    public class ApplicationDbContext : DbContext
+    public class ApplicationDbContext : DbContext, IUnitOfWork
```

Comment:
Rename to QueueReceiverServiceContext to align with Preservation naming standard